app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/infrastruktur/Database.kt

```diff
@@ -267,3 +265,11 @@ class ParameterSetters(
             objectMapper.writeValueAsString<T>(value)
         )
 }
+
+fun <T> ResultSet.getNullableObject(columnLabel: String, type: Class<T>) : T? {
```

Comment:
Jeg er litt usikker på hva tanken bak denne er. Ser hvis jeg bytter ut hele kroppen med bare

```kotlin
return getObject(columnLabel, type)
```
så returneres null noen ganger og alle testene går fortsatt gjennom.

Reply:
Jeg mistenker det er pga hvordan Kotlin håndtere Javas typer (platform types). Så hvis man bytter alle forekomster av `getNullableObject` med `getObject(...)`, så feiler testene, men med `getObject<UUID?>(...)` så går de gjennom.

Reply:
Den exceptionen de nevner i dokumentasjonen tror jeg handler om at andre argument (`type`) bokstavlig talt er `null`.

Reply:
Så tenker kanskje
```kotlin
fun <T> ResultSet.getNullableObject(columnLabel: String, type: Class<T>) : T? =
    getObject(columnLabel, type)
```
er en fin hjelpefunksjon. For hvis man skriver `getObject<UUID?>(...)` i koden, så klager intellij over at det er unødvendig å spesifisere typen.

Reply:
grunnen til at jeg laget denne var at det feilet å gjøre getObject når det ikke finnes noen resultater i join. Da kastes exception.

Reply:
klarer ikke å gjenskape feilen nå, så da var det kanskje noe annet som var galt. Da er denne metoden overflødig og kan fjernes

Reply:
fjernet ae2c96f